rir/src/compiler/pir/instruction.cpp

```diff
@@ -446,5 +460,22 @@ void CallImplicit::printArgs(std::ostream& out, bool tty) {
 
 FrameState* Deopt::frameState() { return FrameState::Cast(arg<0>().val()); }
 
+void Checkpoint::printArgs(std::ostream& out, bool tty) {
+    FixedLenInstruction::printArgs(out, tty);
+    out << " -> BB" << bb()->trueBranch()->id << " (by default) | BB"
+        << bb()->falseBranch()->id << " (if coming from expect)";
+}
+
+BranchInstruction* BranchInstruction::CastBranch(Value* v) {
```

Comment:
I think the verifier still has the explicit list of branch instructions instead of using this interface